src/browser/provider/built-in/phantomjs.js

```diff
@@ -0,0 +1,47 @@
+import { create as createPhantom } from 'phantom';
+
+export default {
+    phantom: null,
+
+    openedPages: {},
```

Comment:
`listAvailableOptionalBrowserNames` nooo
They are not 'optional'. Like if you could go with just `testcafe saucelabs`.
Mmmm.... have to think about it